CIP-0078/CPS-0078.md

```diff
@@ -0,0 +1,41 @@
+---
+CPS: 78
```

Comment:
```suggestion
CPS: ?
```
In the meantime, can you remove the CPS number `78` from the header, filename, and content folder? ... particularly because the next CPS number so far is `2`, and generally because editors should assign these numbers during triage to prevent collisions.

Reply:
Fixed